cli/lib/command/internal_command_settings.dart

```diff
@@ -4,17 +4,67 @@ import 'package:stax/context/context.dart';
 import 'package:stax/settings/settings.dart';
 
 class InternalCommandSettings extends InternalCommand {
+  static const availableSettings = {
```

Comment:
There is no reason for `static const` here.
```suggestion
  static const availableSettings = {
```